package.json

```diff
@@ -1,25 +1,50 @@
 {
   "name": "input-area",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Jupyter Input Area",
   "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "dependencies": {
+    "codemirror": "^5.7.0",
+    "diff-match-patch": "^1.0.0",
+    "phosphor-messaging": "^1.0.5",
+    "phosphor-properties": "^2.0.0",
+    "phosphor-signaling": "^1.1.2",
+    "phosphor-widget": "^1.0.0-beta"
+  },
+  "devDependencies": {
+    "nodemon": "^1.8.0",
+    "rimraf": "^2.4.3",
+    "steal": "^0.12.3",
```

Comment:
The example uses steal.  Should we put that in a separate package json in the examples directory?